### PR TITLE
feat: add multiline string support

### PR DESCRIPTION
**Pull request by iamsauravsharma**

### CODE REVIEW

File: tests/main.test.ts [hunk 1]
```diff
@@ -27,14 +28,14 @@ test("test if env file value matched with actual value", async () => {
...
+        throw new Error("Size of two map is not same");
     }
     for (const [key, val] of expectedMap) {
         const actualValue = actualMap.get(key);
         if (actualValue != val) {
-            console.log(`Expected: ${[...expectedMap.entries()]}`);
-            console.log(`Actual: ${[...expectedMap.entries()]}`);
-            throw new Error("Value doesn't matched");
+            console.log(`Expected: ${actualValue}`);
+            console.log(`Actual: ${val}`);
```
Comment (mrrosoff) [inline tests/main.test.ts]: These are backwards. Also the tests are failing, so the replace function must not be working.

**Comment by mrrosoff** [comment]
Thanks for taking a look at this I appreciate it! :)

**Comment by iamsauravsharma** [comment]
@mrrosoff  I have used your technique for quoting only multi line

**Comment by iamsauravsharma** [comment]
https://github.com/marketplace/actions/create-dotenv new version 3.1.0 is release in marketplace with this feature

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/marketplace/actions/create-dotenv">Create dotenv - GitHub Marketplace</a></strong></div><div>GitHub Action to create .env file in repository</div></blockquote>

**Pull request merged**